tests/app/conftest.py

```diff
@@ -138,46 +141,52 @@ def sample_service(notify_db,
     service = Service.query.filter_by(name=service_name).first()
     if not service:
         service = Service(**data)
-        dao_create_service(service, user)
+        dao_create_service(service, user, service_id)
     else:
         if user not in service.users:
             dao_add_user_to_service(service, user)
     return service
 
 
 @pytest.fixture(scope='function')
-def sample_template(notify_db,
-                    notify_db_session,
-                    template_name="Template Name",
-                    template_type="sms",
-                    content="This is a template:\nwith a newline",
-                    archived=False,
-                    subject_line='Subject',
-                    user=None,
-                    service=None,
-                    created_by=None,
-                    process_type='normal'):
+def sample_template(
+    notify_db,
+    notify_db_session,
+    template_name="Template Name",
+    template_type="sms",
+    content="This is a template:\nwith a newline",
+    archived=False,
+    subject_line='Subject',
+    user=None,
+    service=None,
+    created_by=None,
+    process_type='normal',
+    template_id=None
+):
     if user is None:
         user = create_user()
     if service is None:
         service = sample_service(notify_db, notify_db_session)
     if created_by is None:
         created_by = create_user()
-    data = {
-        'name': template_name,
-        'template_type': template_type,
-        'content': content,
-        'service': service,
-        'created_by': created_by,
-        'archived': archived,
-        'process_type': process_type
-    }
-    if template_type in ['email', 'letter']:
-        data.update({
-            'subject': subject_line
-        })
-    template = Template(**data)
-    dao_create_template(template)
+
+    template = Template.query.filter_by(id=template_id).first()
```

Comment:
i'm not really a huge fan of this (and in the same vein the changes to `sample_service`) - there are two options that i think i'd prefer:

1. move this check into the `_create_slow_delivery_notification` fn - so that fn now says
```python
template = Template.query.get(template_id)
if not template:
    create_template(...)
```
2. change the config in the tests to set `FUNCTIONAL_TEST_PROVIDER_SMS_TEMPLATE_ID` to `sample_template.id` or similar

no huge preference, think the second one is nicer but not sure how the code'll look. thoughts?